exp/performance.md

```diff
@@ -0,0 +1,52 @@
+## Hardware and network setup
+
+See [Requirements](https://discourse.ubuntu.com/t/installation-requirements/17734) for the minimum hardware requirements for Anbox Cloud. Note that these list the minimum requirements, and using more powerful hardware will increase performance.
+
+The overall performance depends not only on the hardware used for the actual Anbox Cloud deployment, but also on the setup used for other components that Anbox Cloud relies on. For example, the etcd database must use a hard disk that is fast enough; see [Hardware recommendations](https://etcd.io/docs/v3.5/op-guide/hardware/) for detailed information.
+
+Also make sure that there is a stable network connection between the nodes of your cluster, to decrease the latency between nodes.
+
+<a name="startup-time"></a>
+## Container startup time
+
+A very noticeable performance issue is a long wait time when starting an application.
+
+When a user starts an application, Anbox Cloud retrieves the application image and launches a new container for it. By default, Anbox Cloud turns off image compression in LXD when launching a container from an image. This method speeds up the launch of the container (because the image does not need to be uncompressed), but it causes more traffic over the network (because the image is transferred uncompressed). If the network connection between your cluster nodes is rather slow, the overall container startup time might improve by enabling image compression. You can change the default configuration by setting the [images_compression_algorithm](https://charmhub.io/ams-lxd/configure#images_compression_algorithm) configuration on the `ams-lxd` charm.
+
+Another configuration that affects the container startup time is [shiftfs_enabled](https://charmhub.io/ams-lxd/configure#shiftfs_enabled). This configuration is currently disabled by default, because it can cause issues with some Android applications. However, if your applications run fine with `shiftfs_enabled` set, it can considerably improve the container startup time. You should be aware though that support for shiftfs might be dropped in future releases.
+
+You should also check the hooks that you use in your application. If you use any startup hooks (`pre-start` or `post-start`) that take a long time or wait for resources to become available, the container startup is delayed. If you use a `post-stop` hook that prolongs the container shutdown, this might also affect the startup time of new containers (because it might not be possible to start more containers until the existing containers terminate).
+
+<a name="client-devices"></a>
+## Client devices
+
+In addition to optimising the performance of your Anbox Cloud deployment, you must also make sure that the client devices that access it can fully utilise its capabilities.
+
+All client devices that access an Anbox Cloud deployment must be capable of low-latency video decoding. They must also use a compatible version of Android WebView.
```

Comment:
Can we add information on what WebView version is required? I know we list the version we use in the release notes, but clients probably don't require that exact version, or do they?

Reply:
As we would like to embrace the latest security update + performance gain from google bring into webview for client devices, the latest stable release, the better. If it's not the case, webview version 90 and higher is our recommendation for clients to install to achieve stable/well performed streaming.
@morphis does version 90 sound aggressive? According to https://chromereleases.googleblog.com/2021/03/
It's been 15 months since the Chrome Android 90.0.xx  was released.

Reply:
>= 90 sounds good to me.